### PR TITLE
Fix for mail notification interval and accept multiple recipients

### PR DESCRIPTION
**Pull request by priyankaborawake**
Signed-off-by: priyanka.borawake <66418474+priyankaborawake@users.noreply.github.com>

# Problem Statement
- Fix for mail notification interval and accept multiple recipients

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by kulrk — APPROVED** [review]

**Comment by priyankaborawake** [comment]
[collections_log.txt](https://github.com/Seagate/cortx-test/files/8725900/collections_log.txt)

File: commons/mail_script_utils.py [hunk 1]
```diff
@@ -57,7 +57,6 @@ def send_mail(self,subject, body):
         msg["Subject"] = subject
         msg["From"] = self.sender
         msg["To"] = self.receiver
-        LOGGER.info("Sending mail :")
-        LOGGER.info("Subject : %s, Body : %s",subject,body)
+        LOGGER.info("Sending mail with Subject %s:",subject)
```
Comment (sonalk0209) [inline commons/mail_script_utils.py:60]: space between , and subject
please check wherever applicable in file

File: libs/iostability/iostability_lib.py [hunk 2]
```diff
@@ -132,7 +132,11 @@ def run(self):
             subject = f"Test {self.test_id} in progress on {self.health_obj.hostname}"
             body = f"hctl Status: {status} \n"
             self.mail_obj.send_mail(subject=subject, body=body)
-            time.sleep(self.interval * 60 * 60)
+            current_time = time.time()
+            while time.time() < current_time + self.interval * 60 * 60:
+                if self.event_pass.is_set() or self.event_fail.is_set():
+                    break
+                time.sleep(60)
```
Comment (sonalk0209) [inline libs/iostability/iostability_lib.py:139]: can we take from yaml?

Comment (priyankaborawake) [reply libs/iostability/iostability_lib.py:139]: This is used for polling if event if set. We can keep it constant.

**Review by sonalk0209 — APPROVED** [review]

**Review by ravindrachoudhari — APPROVED** [review]

**Pull request merged**